forked/1-calico.ps1

```diff
@@ -16,29 +14,38 @@ if (-not(Test-Path -Path $env:KUBECONFIG -PathType Leaf)) {
     exit 1
 }
 
-Write-Output "ABOUT TO RUN install-calico.ps1 script !!!!!!!!!!!!!!!!!!!"
 if (-not (Test-Path -Path ./install-calico.ps1 -PathType Leaf)) {
     Write-Output "WARNING WARNING WARNING : I DONT SEE THE INSTALL-CALICO.ps1 FILE !!!!!!!!!!"
     ls ./
     exit 100
-} else {
-    ### Replace calico scripts with forked ones:
-    Write-Output "Copying calico files............."
-    
-    cp C:/forked/install-calico.ps1 ./
-    cp C:/forked/calico.psm1 ./libs/calico/
-    cp C:/forked/node-service.ps1 ./node/node-service.ps1
-    cp C:/forked/config.ps1 ./
-    Write-Output "................ DONE Copying forked calico files"
-
-    ### This /opt/cni/bin directory is read by the calico node startup script
-    Write-Output "............ RUNNING INSTALL_CALICO.ps1 with the following ENV VARS ............"
-    Get-ChildItem env: 
-    Get-Date
-    .\install-calico.ps1
 }
 
-Write-Output "DONE INSTALLING CALICO I THINK !"
+## ------------------------------------------
+Write-Output "Running install-calico.ps1 script"
+## ------------------------------------------
+
+## Node-service can be replaced on 3.20.1 --
+cp C:/forked/node-service.ps1 ./node/node-service.ps1
```

Comment:
3.20.1 will contains the VXLAN fix - for now copying the node-service to have VXLAN on non-primary interface.